README.md

```diff
@@ -32,7 +32,6 @@ Some configuration options can be fetched from OpenShift if there are correctly
 Tools can be deployed by using `overlays/kuadrant` overlay and deploying RHSSO with the provided script like this:
```

Comment:
that ` and deploying RHSSO with the provided script like this` piece is redundant now and should also be removed.

Reply:
Agreed!